code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1522,6 +1526,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 							break
 			if((!target_table && !target_collateral_human && !target_disposal_bin && !target_pool && !target.IsKnockdown()) || directional_blocked)
 				target.Knockdown(SHOVE_KNOCKDOWN_SOLID)
+				target.Immobilize(SHOVE_KNOCKDOWN_SOLID)
```

Comment:
I think the immobilization should be shorter, maybe like 1 to 2 seconds instead of the full knockdown length.

Reply:
Oh fuck, I never pushed that. I already changed it to be this.